content/docs/1.8.0/advanced-resources/backing-image/backing-image.md

```diff
@@ -154,6 +159,32 @@ Since v1.3.0, users can download existing backing image files to the local via U
 #### Notice:
 - Users need to make sure the backing image existence when they use UI to create or restore a volume with a backing image specified.
 - Before downloading an existing backing image file to the local, users need to guarantee there is a ready file for it.
+- Downloading of V2 Data Engine backing images is currently not supported.
+
+## Create a V2 Data Engine Backing Image
+
+Starting v1.8.0, you can create a backing image that is supported by the V2 Data Engine by configuring `Data Engine` in the YAML (through the UI or a StorageClass).
+
+### Parameters During Creation
+
+All parameters are the same as that of the V1 Data Engine backing image, except for `Data Engine`.
+
+#### Backing Image Data Sources
+
+You can prepare a V2 Data Engine backing image using any of the supported data sources.
+- Download a backing image file (using a URL).
+- Upload a file from your local machine. This option is available to Longhorn UI users.
+- Export an existing in-cluster V1 Data Engine volume as a backing image.
+- Restore a backing image from the backupstore. For more information, see [Backing Image Backup](../backing-image-backup).
+- Clone a V1 backing image.
+
+#### Notice
+
+- The following operations are currently not supported:
+  - Exporting from a V2 Data Engine volume
+  - Cloning a V2 backing image
+  - Backing up a V2 backing image
+- Because of a V2 Data Engine limitation, Longhorn first converts a qcow2 image to a raw image, and then stores the data in a V2 Data Engine backing image.
```

Comment:
Can we briefly explain the limitation or reference the corresponding issue for the limitation?

Reply:
Added
> Unlike the V1 Data Engine, which is file-based, the V2 Data Engine requires Longhorn to store the backing image data in an SPDK logical volume. As a result, for qcow2 images, Longhorn must first convert the qcow2 image to a raw format before storing the data to the V2 Data Engine backing image, enabling it to read the correct data.